### ISSUE TITLE
Suffix/prefix aria-labelledby is required but can be left empty

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2344]
It's currently possible to build an input with a suffix or prefix, but without setting the `id` to be used by the input's `aria-labelledby` attribute.  Conditions should be set in the macro so the component can't be built without it.
